### DISCUSSION TITLE
Is it possible to do server-side calls to a standalone trpc server?

### DISCUSSION THREAD

**Discussion by thereis** [trpc/trpc#5989] [Q&A]
In the examples that you guys have about using trpc with server-side prefetch/hydration, I noticed that we need the app router in order to have the access to the `prefetch` method and `HydrateClient` component as well:

https://github.com/trpc/examples-next-app-dir/blob/main/src/trpc/rq-server.ts#L28

Is it possible to create a caller that points to an external tRPC server (preferably by URL) which exports the same api such as `prefetch` and `HydrateClient`?

**Comment by davidroulier** [comment]
Hello,

Any update on this ?

**Reply by KATT** [reply]
We should allow this or make support for it, feel free to open a feature request
